docs/platform/server-v3/modules/about.md

```diff
@@ -14,100 +14,132 @@ enterprise_only: true
 - [WordPress module (blog)](../../integration/wordpress)
 - [Algolia search module (search)](../../integration/algolia)
 
-## Old vs new approach
+## What is a module in Falcon
 
-In previous version of Falcon Server (version 2) there was a distinct separation between extensions, api clients, rest endpoint handlers and event handlers. Each of these needed to be implemented and loaded as a separate package. Falcon Server 3 changes that approach and introduces concept of a module. 
+Falcon Module is a package that contains implementations of all the pieces required to achieve particular thing. Falcon Module needs to exports their content via named export, default export will be ignored. Falcon Server expects only known Module extension classes, anything else, which is outside of the predefined list will be ignored:
 
-## What is a module in Falcon
+- [Data Source](./common-services/data-sources) - Used by GraphQL resolvers to fetch the data from various services. By default, Falcon Module can contain only one implementation of it due to auto-discovery. However, by using our Dependency Injection framework, you can define more of them, please read more about that in [Falcon Module and Dependency Injection](#falcon_module_and_dependency_injection) section.
+- [Rest Endpoint Handler](./common-services/rest-endpoints) - REST webhook handler allows you to execute any action for incoming HTTP request. Module can contain any number of it.
+- [Event Handler](./common-services/event-handlers) - Handler for in-proc emitted events. Module can contain any number of it.
```

Comment:
```suggestion
- [Event Handler](./common-services/event-handlers) - Handler for in-proc emitted events. Module can contain any number of these.
```

Reply:
`for in-proc emitted` - perhaps we can explain these. I know they are events that our middleware can use. But I've never heard the expression "in-proc". Is that a common term?